src/components/UserDashboard/TwitterBlock.js

```diff
@@ -0,0 +1,127 @@
+import React, { Component } from 'react';
+import styled from 'react-emotion';
+import scriptjs from 'scriptjs';
+import Spinner from 'react-spinkit';
+import { withTheme } from 'emotion-theming';
+
+import Card from 'uikit/Card';
+import ExternalLink from 'uikit/ExternalLink';
+import { Box } from 'uikit/Core';
+
+const StyledCard = styled(Card)`
+  margin: 15px 0 15px 40px;
+  padding: 21px 29px 29px 27px;
+
+  flex-direction: column;
+
+  & #twitter-timeline {
+    width: 100%;
+    height: 100%;
+    padding-bottom: 66px;
+  }
+`;
+
+class TwitterBlock extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      loaded: false,
+    };
+  }
+
+  componentDidMount() {
+    const timelineEl = document.getElementById('twitter-timeline');
+    const handle = this.props.handle;
+
+    scriptjs('https://platform.twitter.com/widgets.js', () => {
+      const twttr = window.twttr;
+
+      twttr.ready(() => {
+        twttr.widgets
+          .createTimeline(
+            {
+              sourceType: 'profile',
+              screenName: handle,
+            },
+            timelineEl,
+            {
+              chrome: 'nofooter noheader',
+              height: '100%',
+            },
+          )
+          .then(() => {
+            twttr.widgets.load(timelineEl);
+            this.setState({ loaded: true });
+          });
+      });
+    });
+  }
+
+  renderTwitterTitle() {
```

Comment:
This might make more sense to read if it was just another stateless React component that takes `handle` and spit out some jsx